fuel/datasets/iris.py

```diff
@@ -0,0 +1,29 @@
+from fuel.datasets import H5PYDataset
+from fuel.utils import find_in_data_path
+
+
+class Iris(H5PYDataset):
+    u"""Iris dataset.
+    Iris [LBBH] is a simple pattern recognition dataset, which consist of
+    3 classes of 50 examples each having 4 real-valued features each, where
+    each class refers to a type of iris plant. It is accessible through the
+    UCI Machine Learning repository [UCI].
+    .. [IRIS] Ronald A. Fisher, *The use of multiple measurements in
+       taxonomic problems*, Annual Eugenics, 7, Part II, 179-188,
+       September 1936.
+    .. [UCI] https://archive.ics.uci.edu/ml/datasets/Iris
+    Parameters
+    ----------
+    which_sets : tuple of str
+        Which split to load. Valid values are 'train' and 'test',
+        corresponding to the training set (50,000 examples) and the test
+        set (10,000 examples).
```

Comment:
Could you change the description to reflect my earlier comment on the `'all'` split?